cmake/Modules/CocosConfigDefine.cmake

```diff
@@ -166,19 +166,6 @@ endfunction()
     if(MSVC)
         target_compile_options(${target}
             PUBLIC /MP
-            PUBLIC /Z7
```

Comment:
the reason to add `/Z7` is `too large PDB file` occurs to me when compile `cpp-tests` project

Reply:
/Z7 it is old debug format.
All CI tests passed

Reply:
Yeah this has been discussed, but it may not necessary for VS2017+ due to the improvements in the debugging database system(s). I'm not sure the best course of action here though, but maybe if you can add that option only to cpp-tests then that would be good enough since others can choose to modify their own project target options?

Reply:
@stevetranby idea of this change is that common cmake configuration does not force settings on user. If user really want to change something it possible from his code without changing cocos code. It is bad cmake practice to set compiler flags https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#get-your-hands-off-cmake_cxx_flags 

About visual studio debug database (pdb files): from my experience it does not matter with or without /Z7 pdb become broken from time to time ;)

Reply:
Sounds good. Makes sense to give control to user. 

Maybe best to just document in cpp-tests that one can try no debug (??option??) or light debug (/Z7) if they have issues building.

I am not sure if the new templated CCActions helped or hurt in that regard, but I remember the "PDB too large" issue for me was related to the large amount of code and derived classes for all the actions. 

(It would be great to see the action system refactored into a "few lines of code" generic approach by passing in easing function instead of having the large hierarchy of derived classes - even if now templated).

I digress :-]